lib/modules/manager/composer/artifacts.spec.ts

```diff
@@ -149,6 +156,61 @@ describe('modules/manager/composer/artifacts', () => {
     expect(execSnapshots).toMatchSnapshot();
   });
 
+  it('git-tags hostRule for github.com set github-token in COMPOSER_AUTH', async () => {
+    hostRules.add({
+      hostType: PlatformId.Github,
+      matchHost: 'api.github.com',
+      token: 'ghs_token',
+    });
+    hostRules.add({
+      hostType: GitTagsDatasource.id,
+      matchHost: 'github.com',
+      token: 'ghp_token',
+    });
+    fs.readLocalFile.mockResolvedValueOnce('{}');
+    const execSnapshots = mockExecAll(exec);
+    fs.readLocalFile.mockResolvedValueOnce('{}');
+    const authConfig = {
+      ...config,
+      registryUrls: ['https://packagist.renovatebot.com'],
+    };
+    git.getRepoStatus.mockResolvedValueOnce(repoStatus);
+    expect(
+      await composer.updateArtifacts({
+        packageFileName: 'composer.json',
+        updatedDeps: [],
+        newPackageFileContent: '{}',
+        config: authConfig,
+      })
+    ).toBeNull();
+    expect(execSnapshots).toMatchSnapshot();
```

Comment:
use `toMatchObject` instead

Reply:
I updated my tests an and the one above to match only ony the content (and existence) of the COMPOSER_AUTH environment variable.  

I did not use `toMatchObject` to avoid matching the whole exec which sound irrelevent to me in the context of this specific test.

Reply:
oh I missed this comment before askig re-review because the conversation was resolved.  I will try again, toMatchObject did not seem to match partial objects when I tried.

Reply:
it will, but the array length has to match

Reply:
Ok it's done, I don't know why it did'nt work before but I now use `toMatchObject()`